File: benchmark/PHP 7.0/函数调用速度测试.php
<?php

function fib(int $i, int $j):int
{
	return $i+$j;
}

$starttime = microtime(true);

$sum = 0;
for ($i = 0; $i < 1000*10000; $i++)
	$sum += fib($i, $i+1);

echo $sum, "\n";

echo microtime(true)-$starttime."\n";

File: benchmark/Lua 5.3/函数调用速度测试.lua
function fib(i, j)
	return i+j
end

local start = os.clock()

local sum = 0
for i = 0, 1000*10000-1, 1 do
	sum = sum + fib(i, i+1)
end
print(sum)

print(os.clock()-start, "\n")
